test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -154,6 +226,58 @@ def _run_test(m):
 
         _run_test(model)
 
+    def run_model(self, model, inputs_list, tolerate_small_mismatch=False, do_constant_folding=True, dynamic_axes=None,
```

Comment:
do we need to add those two additional functions to our test folder? It looks like there is a lot of duplicated code.
Can we change the existing functions?

Reply:
This is a very good point. These two additional functions are from torch.vision CI, therefore the current pytorch CI code has minimal code change from torch.vision CI, so it can correctly reflect all CI test from torch.vision. Refactoring code sometimes is pain, because the comparison method is somehow different, so it is not very easy to merge existing functions, and error prone. I would prefer to leave it as for now.

Reply:
I agree with Ksenija. It is better to clean up the additional functions, and maybe factor out the parts of the comparison code that is different.

Reply:
Done.